### PR TITLE
chore(code-gen-types): add new dotnet event for appsettings.json

### PR DESCRIPTION
**Pull request by overbit**
Close: #8509

## PR Details

Add new dotnet event for `appsettings.json` generation

## PR Checklist

- [ ] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: libs/util/code-gen-types/src/dotnet-plugin-events-params.types.ts [hunk 1]
```diff
@@ -86,10 +86,6 @@ export type VariableDictionary = {
   [variable: string]: string;
 }[];
 
-export interface CreateServerDotEnvParams extends EventParams {
-  envVariables: VariableDictionary;
-}
-
```
Comment (abrl91) [inline libs/util/code-gen-types/src/dotnet-plugin-events-params.types.ts:92]: Why did you remove it?

File: libs/util/code-gen-types/src/dotnet-plugin-events-params.types.ts [hunk 2]
```diff
@@ -114,6 +110,10 @@ export interface CreateServerCsprojParams extends EventParams {
     privateAssets?: string;
   }[];
 }
+export interface CreateServerAppsettingsParams extends EventParams {
+  fileContent: string;
+  updateProperties: { [key: string]: any }[];
```
[the remark this answers is not in the document]
Comment (overbit) [reply libs/util/code-gen-types/src/dotnet-plugin-events-params.types.ts:115]: Great idea!

[the remark this answers is not in the document]
Comment (overbit) [reply libs/util/code-gen-types/src/dotnet-plugin-events-params.types.ts]: done

**Review by abrl91 — APPROVED** [review]
